src/internal/packager2/load.go

```diff
@@ -36,6 +37,8 @@ type LoadOptions struct {
 	SkipSignatureValidation bool
 	Filter                  filters.ComponentFilterStrategy
 	Output                  string
+	// [ALPHA] Namepsace override for package deployment
+	NamespaceOverride string
```

Comment:
This should be a parameter on `cluster.GetDeployedPackage` or maybe that function should get it's own opts struct. Since namespaceOverrides is only relevant to cluster pulls we shouldn't make it an option here.

Reply:
Agreed - thought about options more but I didn't want to obfuscate LoadOptions or embed them if possible.